ebms-send-in/src/test/kotlin/no/nav/emottak/fellesformat/FellesFormatWrapperTest.kt

```diff
@@ -21,6 +33,43 @@ class FellesFormatWrapperTest {
     fun `Wrap HarBorgerFrikort in Fellesformat`() {
         val sendInRequest = validSendInHarBorgerFrikortRequest.value
         val fellesFormat = sendInRequest.asEIFellesFormat()
+        val payload = sendInRequest.payload
+        val msgHead = unmarshal(String(payload), MsgHead::class.java)
+
+        val action = msgHead.document.map { doc -> doc.refDoc.content.any }.first().first()
+        val refParam = refParamFrikort(action)
+        assertEquals("123456", birthDay(refParam))
+        log.info(sendInRequest.marker(), "refParam ${birthDay(refParam)}")
+        Assertions.assertEquals(fellesFormat.mottakenhetBlokk.ebService, sendInRequest.addressing.service)
+        log.info(marshal(fellesFormat))
+    }
+
+    @Test
+    fun `Wrap HarBorgerEgenadelMengde in Fellesformat`() {
+        val sendInRequest = validSendInHarBorgerFrikortMengdeRequest.value
+        val fellesFormat = sendInRequest.asEIFellesFormat()
+        val payload = sendInRequest.payload
+        val msgHead = unmarshal(String(payload), MsgHead::class.java)
+
+        val action = msgHead.document.map { doc -> doc.refDoc.content.any }.first().first()
+        val refParam = refParamFrikort(action)
```

Comment:
Funksjonene for referanseparameter og validering av outputen kan gjerne flyttes til egne tester.